src/extension.ts

```diff
@@ -105,9 +105,33 @@ export async function activate(context: vscode.ExtensionContext) {
 		}
 
 		function parseAndDecorate(): MessageList {
-			let [messageList, errors] = parser.parse(editor!.document.getText(), l10nOptions)!;
+			let templatePathFromOptions = l10nOptions?.['template-arb-file'];
+			let templateMessageList: MessageList | undefined;
+			let templateErrors: Literal[] | undefined;
+			let [templatePathFromFile, messageList, errors] = parser.parse(editor!.document.getText(), l10nOptions)!;
+			if (templatePathFromOptions || templatePathFromFile) {
+				let templatePath: string;
+				if (templatePathFromOptions) {
+					if (path.isAbsolute(templatePathFromOptions!)) {
+						templatePath = templatePathFromOptions!;
+					} else {
+						templatePath = path.join(path.dirname(l10nYamlPath!), templatePathFromOptions);
+					}
+				} else {
+					if (path.isAbsolute(templatePathFromFile!)) {
+						templatePath = templatePathFromFile!;
```

Comment:
I think this means the path from options will be used in preference to the one in the file - should it be the other way around? (eg. you can set a global one in options, but override it per-file)?

Reply:
That was my thinking, but all these `templateFile...` variables got me confused :laughing: